src/cli/configure/upgrade/index.test.ts

```diff
@@ -0,0 +1,138 @@
+import type { Dirent } from 'fs';
+import { readdir, writeFile } from 'fs/promises';
+
+import type { NormalizedPackageJson } from 'read-pkg-up';
+
+import { getConsumerManifest } from '../../../utils/manifest';
+import { getSkubaVersion } from '../../../utils/version';
+
+import { upgradeSkuba } from '.';
+
+jest.mock('../../../utils/manifest');
+jest.mock('../../../utils/version');
+jest.mock('fs/promises');
+jest.mock('../../../utils/logging');
+
+beforeEach(() => {
+  jest.clearAllMocks();
+});
+
+describe('upgradeSkuba', () => {
+  it('should throw an error if no skuba manifest can be found', async () => {
+    jest.mocked(getConsumerManifest).mockResolvedValue({
+      packageJson: {
+        _id: 'test',
+        name: 'some-api',
+        readme: '',
+        version: '1.0.0',
+      } as NormalizedPackageJson,
+      path: '/package.json',
+    });
+
+    await expect(upgradeSkuba()).rejects.toThrow(
+      'Could not find a skuba manifest, please run `skuba configure`',
+    );
```

Comment:
Is this a desirable breaking change? If so, should we warn and skip patching until the next major version?

One consideration is that this may move us further away from #603. I like zero-config support as an aspiration, but appreciate that we'd have to evaluate the tradeoffs to get us there.

Reply:
I suppose we only require a simple `{ skuba: { version: string } }` field. Maybe we can just auto configure the bare minimum?